src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -358,6 +358,7 @@ public static class Pipeline
                 //
                 // Keep alphabetical. If you add or remove a variable here, do the same in ReadOnlyVariables
                 //
+                public static readonly int RetriesOnFailure = 3;
```

Comment:
This isn't the appropriate place for this constant. The constants in `Constants.Variables` are the names of actual Pipelines variables, not variables in the general sense. I think defining a constant within `TFCliManager` would be reasonable, since this is specific to that class and its subclasses and it wouldn't require changing the project dependencies around.

Reply:
Sorry, I meant to say ` TfsVCCliManager`, not `TFCliManager`, would be a reasonable place for the constant.